gensrc/thrift/DataSinks.thrift

```diff
@@ -314,13 +330,12 @@ struct THivePartition {
 struct THiveTableSink {
     1: optional string db_name
     2: optional string table_name
-    3: optional list<string> data_column_names
-    4: optional list<string> partition_column_names
-    5: optional list<THivePartition> partitions
-    6: optional list<THiveBucket> buckets
-    7: optional PlanNodes.TFileFormatType file_format
-    8: optional THiveCompressionType compression_type
-    9: optional THiveLocationParams location
+    3: optional list<THiveColumn> columns
+    4: optional list<THivePartition> partitions
+    5: optional THiveBucket bucket_info
+    6: optional PlanNodes.TFileFormatType file_format
+    7: optional THiveCompressionType compression_type
+    8: optional THiveLocationParams location
```

Comment:
Why here is a `THiveLocationParams`?
There is already a `THiveLocationParams` in `THivePartition`

Reply:
> Why here is a `THiveLocationParams`? There is already a `THiveLocationParams` in `THivePartition`

Table level and partition level.